WebSTR/templates/downloads.html

```diff
@@ -10,7 +10,11 @@ <h1>Data sources for hg38</h1>
     <div class="row">
 <ul>
 <li>Full SQL dump of hg38 data to set up WebSTR-API locally <a href="https://drive.google.com/file/d/1oF2G-4Q-rYoq2SV2mN8zo9s1SkRFHG0S/view?usp=sharing">webstr_2023_06_23.dump</a></li>
-<li>EnsemblTR Reference panel archive <a href="https://drive.google.com/file/d/1hicWMyZKpSvdz4fa-g-POBRfUY6iPVam/view?usp=drive_link">ensembletr.zip</a></li>
+<li>EnsemblTR Reference panel archive <a href="https://drive.google.com/file/d/1hicWMyZKpSvdz4fa-g-POBRfUY6iPVam/view?usp=drive_link">ensembletr.zip</a>
+  <ul>
+    <li>EnsembleTR individual-level genotypes and population-level summary statistics cna be downloaded from <a href=" https://github.com/gymrek-lab/EnsembleTR">https://github.com/gymrek-lab/EnsembleTR</a></li>
```

Comment:
minor typo: cna->can
Also I would not have as a nested list (so can remove the <ul> and </ul> around this bullet point